components/analytics/components/Summary/index.js

```diff
@@ -1,14 +1,69 @@
 import React from 'react';
 import { connect } from 'react-redux';
 import get from 'lodash/get';
+import { Row, Col } from 'antd';
 import PropTypes from 'prop-types';
+import { css } from 'emotion';
 import { getAppAnalyticsSummaryByName } from '../../../../modules/selectors';
 import { getAppAnalyticsSummary } from '../../../../modules/actions';
 import Loader from '../../../shared/Loader/Spinner';
 import { displayErrors } from '../../../../utils/heplers';
-import SummaryCard from './SummaryCard';
 import Flex from '../../../shared/Flex';
 
+const cardStyle = css`
+	margin-bottom: 8px;
+	background: rgba(229, 230, 233, 0.21);
+	padding: 15px 0;
+	p,
+	h2 {
+		margin: 0;
+		text-align: center;
+	}
+
+	p {
+		font-size: 1em;
+		font-weight: bold;
+		color: #8c8c8c;
+	}
+
+	h2 {
+		color: #595959;
+		display: flex;
+		justify-content: center;
+		vertical-align: middle;
+	}
+
+	h2 span {
+		color: #8c8c8c;
+		font-weight: bold;
+		margin-left: 5px;
+		font-size: 16px;
+	}
+`;
+
+const cardContainer = css`
+	padding: 10px;
+`;
+
+const SummaryCard = ({
```

Comment:
@jyash97 
1. Why did you not define the component in the separate file, we should not keep two components in the same file?
2. Ideally, you should have modified the existing `SummaryCard` component.